config/http.conf

```diff
@@ -2,12 +2,19 @@ http {
   include /etc/nginx/mime.types;
   include /etc/nginx/log.conf;
 
-  server_tokens         off;
-  keepalive_timeout     20s;
-  sendfile              on;
-  tcp_nopush            on;
-  client_max_body_size  400m;
-  client_body_timeout   300s;
+  server_tokens           off;
+  sendfile                on;
+  tcp_nopush              on;
+
+  client_max_body_size    500m;
+  client_body_buffer_size 128k;
+  client_body_timeout     300s;
```

Comment:
Wondering if we should align the [`client_body_timeout`](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_timeout) value with the `client_header_timeout`?

Reply:
No objections to that. Should probably have.